runtime/src/accounts_background_service.rs

```diff
@@ -579,8 +577,7 @@ impl AccountsBackgroundService {
     pub fn setup_bank_drop_callback(bank_forks: Arc<RwLock<BankForks>>) -> DroppedSlotsReceiver {
         assert_eq!(bank_forks.read().unwrap().banks().len(), 1);
 
-        let (pruned_banks_sender, pruned_banks_receiver) =
-            crossbeam_channel::bounded(MAX_DROP_BANK_SIGNAL_QUEUE_SIZE);
+        let (pruned_banks_sender, pruned_banks_receiver) = crossbeam_channel::unbounded();
```

Comment:
With this change, the try_send here can be removed as well, and the `BANK_DROP_QUEUE_STATS` will be removed: https://github.com/solana-labs/solana/blob/3f4731b37ff54a444d378a1ee7049cce1f0d4f75/runtime/src/accounts_background_service.rs#L127-L133

Would be good to be able to monitor the size of this queue in some other way, maybe on the receiving end and flag when it gets too large.

Reply:
updated.

Reply:
I pushed a change to report the length the pruned bank channel. When the length is above the current threshold, i.e. 10K, it will report a warning data point at the interval of 1 minute.